source/upgrade.txt

```diff
@@ -0,0 +1,156 @@
+.. _java-upgrade-driver:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+In this section, you can identify the changes you need to make to your
+application to upgrade your driver to a new version or to transition from
+the legacy API to the current API.
+
+When you upgrade your driver version or use multiple versions of the
+driver, if there are any breaking changes between them that you have not
```

Comment:
It's not possible to use multiple versions of the driver in the same running application (unless you do some crazy class loader things)

Reply:
That's not the information I was trying to convey here, so I'm guessing I need it to make it more clear or maybe just omit that info and let the smaller audience to which it could apply infer it.

Reply:
Can you describe more what you were trying to convey?

Reply:
I guess this was removed?

Reply:
Yes, this was a cross-thread communication:
https://github.com/mongodb/docs-java/pull/212#discussion_r824845911

You mentioned that it could be misconstrued, which I agreed with and removed it. The text that replaced it is here:

https://github.com/mongodb/docs-java/pull/212/files#diff-9c988395fdf1141e52aa9fa946e169ad1d9fb7de62fd44fc1b95bc1885b9506dR19-R20